src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/customer/index_response.json

```diff
@@ -0,0 +1,144 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 2,
+    "total": 11,
+    "_links": {
+        "self": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "first": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "last": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        },
+        "next": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": @integer@,
```

Comment:
Hmm, in fact I would remove that check totally (that this is integer, numeric or whatever), cause if you use different database as identifier (ID) you can even get just unique string (like sha256 or similar).